1-mandatory/1-alarmclock/alarmclock.js

```diff
@@ -1,4 +1,21 @@
-function setAlarm() {}
+function setAlarm() {
+  let alarmSetValue = document.getElementById("alarmSet").value;
+  let timeDisplay = document.getElementById("timeRemaining");
+ let interval = setInterval(() => {
+ let minutes =Math.floor(alarmSetValue / 60);
+ let seconds = alarmSetValue%60;
+ minutes = minutes >= 10 ? minutes : "0" + minutes;
+ seconds = seconds >= 10 ? seconds : "0" + seconds;
+timeDisplay.innerText=`Time Remaining: ${minutes}:${seconds}`;
```

Comment:
you could extract this bit of code out into a helper function, maybe called `formatTime` or something like that